.github/workflows/checks.yml

```diff
@@ -0,0 +1,46 @@
+name: Continuous integration
+on: [push]
+jobs:
+  backend:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11.5-alpine
+        ports:
+        - 5433:5432
+    steps:
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6.9
+    - name: Update pip
+      run: python -m pip install --upgrade pip
+    - name: Install tox
+      run: python -m pip install tox
```

Comment:
See: https://docs.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#running-tests-with-tox

Normally GitHub actions runs **jobs** (like the "backend" and "frontend" jobs in this workflow) in parallel but it cannot run steps within a job in parallel. And each job runs in its own isolated GHA "environment" (VM?) so if we were to make separate GHA jobs for formatting, linting, tests, functests and bddtests, then we'd have to install Postgres, Python, checkout the git repo, update pip, build the frontend assets, etc five times over (albeit in parallel but they might throttle us). Instead, the tox command below will use [tox's parallel mode](https://tox.readthedocs.io/en/latest/example/basic.html?highlight=parallel#parallel-mode) to run a bunch of stuff in parallel within a single GHA job.